src/status_im/contexts/communities/events.cljs

```diff
@@ -64,33 +64,56 @@
       (update :chats <-chats-rpc)
       (update :token-permissions seq)
       (update :categories <-categories-rpc)
+      (assoc :membership-permissions?
+             (some #(= (:type %) constants/community-token-permission-become-member)
```

Comment:
This solution here is probably okay for your PR, but as we know @ajayesivan is also working in a PR that is trying to fix all places that need to consume the sequence of token permissions. Perhaps after you merge this one, or if Ajay merges his one first we should try to use a unified predicate to tell if there are membership permissions, just so we avoid bugs because we might forget to update all places with duplicated logic.